tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -14,6 +14,8 @@
 # Where N is the unique instance number for the google-rtc-aec object within the same alsaconf node.
 Define {
 	AEC_UUID	"a6:a0:80:b7:9f:26:6f:46:b4:77:23:df:a0:5a:f7:58"
+	# AEC requires 10ms buffers for processing
+	M		10
```

Comment:
M is too simple, is it duration or period?

Reply:
How about BUFFER_PERIOD_MS?